src/utilities/__tests__/url.test.js

```diff
@@ -75,94 +75,39 @@ describe('url.js', () => {
       }).not.toThrow();
     });
 
-    it("doesn't clarify keyword when it's null", () => {
-      searchVarsClone.keyword = keywords.none;
-      searchVarsClone.category = categoryExamples.none;
-      searchVarsClone.addressLatLng = addressLatLngs.none;
-
-      expect(requestBuilder(searchVarsClone)).toBe(
-        encodeURI(`${API_PATH}datastore_search?distinct=true&resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}`),
-      );
-    });
-
-    it("doesn't clarify keyword when it's less than or equal to 2 chars", () => {
+    it("is not valid when the keyword length is <= 2 chars and there are no other search terms", () => {
       searchVarsClone.keyword = keywords.tooShort;
       searchVarsClone.category = categoryExamples.none;
       searchVarsClone.addressLatLng = addressLatLngs.none;
 
-      expect(requestBuilder(searchVarsClone)).toBe(
-        encodeURI(`${API_PATH}datastore_search?distinct=true&resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}`),
-      );
+      expect(isValidQuery(searchVarsClone)).toBe(false)
     });
 
     it("clarifies keyword when it's greater than 2 chars", () => {
       searchVarsClone.keyword = keywords.valid;
       searchVarsClone.category = categoryExamples.none;
       searchVarsClone.addressLatLng = addressLatLngs.none;
 
-      expect(requestBuilder(searchVarsClone)).toBe(
-        encodeURI(
-          `${API_PATH}datastore_search?distinct=true&resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}&q=${
-            searchVarsClone.keyword
-          }`,
-        ),
-      );
+      expect(requestBuilder(searchVarsClone))
+        .toContain(`q=${searchVarsClone.keyword}`);
     });
 
-    it("doesn't filter wihtout a category", () => {
+    it("doesn't filter without a category", () => {
       searchVarsClone.keyword = keywords.valid;
       searchVarsClone.category = categoryExamples.none;
       searchVarsClone.addressLatLng = addressLatLngs.none;
 
-      expect(requestBuilder(searchVarsClone)).toBe(
-        encodeURI(
-          `${API_PATH}datastore_search?distinct=true&resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}&q=${
-            searchVarsClone.keyword
-          }`,
-        ),
-      );
+      expect(requestBuilder(searchVarsClone))
+        .not.toContain('filters={"LEVEL_1_CATEGORY":');
     });
 
     it('filters by category', () => {
       searchVarsClone.keyword = keywords.valid;
       searchVarsClone.category = categoryExamples.valid;
       searchVarsClone.addressLatLng = addressLatLngs.none;
 
-      expect(requestBuilder(searchVarsClone)).toBe(
-        encodeURI(
-          `${API_PATH}datastore_search?distinct=true&resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}&q=${
-            searchVarsClone.keyword
-          }${categories(searchVarsClone.category)}`,
-        ),
-      );
-    });
-
-    it("doesn't limit when there is no address", () => {
-      searchVarsClone.keyword = keywords.valid;
```

Comment:
This behaviour no longer applies. There used to be an increase on the result limit when doing a location search (to ensure more relevant results were fetched from CKAN?)